pkg/report/github/github.go

```diff
@@ -105,8 +105,25 @@ func (w Writer) Write(ctx context.Context, report types.Report) error {
 		manifest.Name = string(result.Type)
 		// show path for language-specific packages only
 		if result.Class == types.ClassLangPkg {
-			manifest.File = &File{
-				SrcLocation: result.Target,
+			if report.ArtifactType == ftypes.ArtifactContainerImage {
+				// `RepoDigests` ~= <registry>/<image_name>@sha256:<image_hash>
+				// `RepoTag` ~= <registry>/<image_name>:<image_tag>
+				// By concatenating the hash from `RepoDigests` at the end of `RepoTag` we get all the information
+				image_reference := strings.Join(report.Metadata.RepoTags, ", ")
```

Comment:
Can you also update other variables that use `_`?

Reply:
> Also it might make sense to use the separator , (without space) to get a string without spaces.

@Maxim-Durand wdyt about this? Does it make sense?

Reply:
> Can you also update other variables that use _?

Already done in https://github.com/aquasecurity/trivy/pull/5999/commits/fc53d372906b988e1ca60c82ac637df406791d96

> > Also it might make sense to use the separator , (without space) to get a string without spaces.
> 
> @Maxim-Durand wdyt about this? Does it make sense?

I honestly have no opinion on this as having a space there shouldn't break anything, so I left it as it was (but I'm totally ok changing it).

Is there a particular reason you want to remove the space ? If not, I'd say "if it ain't broke, don't fix it".

Reply:
> Already done in https://github.com/aquasecurity/trivy/commit/fc53d372906b988e1ca60c82ac637df406791d96

Thanks!

> I honestly have no opinion on this as having a space there shouldn't break anything, so I left it as it was (but I'm totally ok changing it).
Is there a particular reason you want to remove the space ? If not, I'd say "if it ain't broke, don't fix it".

I was worried that this might break something in GitHub and make it harder to parse that string (I'm not sure of the use case or if this would be needed).
But if you were told that this is normal, let these spaces remain.